main.js

```diff
@@ -102,31 +111,158 @@ function createDetailPage(travelPackage) {
   return $div
 }
 
-//return to home
+//calculate total
+function calculateTotal(cartedTravelPackage) {
+  var total = 0
+  for (var i = 0; i < cartedTravelPackage.length; i++) {
+    total += cartedTravelPackage[i].price
+  }
+  return total
+}
+
+//display total
+function displayTotal() {
+  var $div = document.getElementById('viewtotal')
+  var $span = document.createElement('span')
+
+  $div.innerHTML = ""
+
+  var viewTotal = calculateTotal(app.cart)
+  $span.textContent = "Your Total is $" + viewTotal
+
+  $div.appendChild($span)
+
+  return $div
+}
+
+//create checkout product
+function createCheckoutProduct(cartedTravelPackage) {
+  var $div = document.createElement('div')
+  var $img = document.createElement('img')
+  var $location = document.createElement('span')
+  var $price = document.createElement('span')
+  var $date = document.createElement('input')
+
+  $($date).daterangepicker()
+
+  $div.classList.add('row')
+
+  $img.setAttribute('src', cartedTravelPackage.img)
+  $img.classList.add('col-md-3', 'img-responsive')
+
+  $location.classList.add('div', 'col-md-6')
+
+  $price.classList.add('div', 'col-md-6')
+
+  $location.textContent = cartedTravelPackage.destination
+  $price.textContent = '$' + cartedTravelPackage.price
+
+  $date.id = 'daterange'
+
+  $div.appendChild($img)
+  $div.appendChild($location)
+  $div.appendChild($price)
+  $div.appendChild($date)
+
+  return $div
+}
+
+//render checkout product
+function renderCheckoutProduct() {
```

Comment:
Try to keep functions pure when possible, this function should return something instead of updating the DOM.